Resources/public/css/views/dashboard/dashboard.css

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview * {
+    box-sizing: border-box;
+}
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview > .pure-u {
```

Comment:
no you remove the `pure-u` class which is not supposed to be used outside of a `pure-g` (in that sense it's a hack). I guess the margin issue actually comes from the top margin of the `h1`, so normally you can fix that way.